libyara/modules/pe.c

```diff
@@ -1961,6 +1999,7 @@ begin_declarations;
   declare_function("imports", "s", "i", imports_dll);
   declare_function("locale", "i", "i", locale);
   declare_function("language", "i", "i", language);
+  declare_function("rich_version", "i", "i", rich_version);
```

Comment:
I implemented your suggestions and I also added a `toolid` function. Checking ID/version pairs can also be done, but how would that function look? `rich_signature.toolversion` maybe? I thought about optional parameters too. Maybe something like `rich_signature.version(version, toolid [optional)` and `rich_signature.toolid(toolid, version [optional]`?